Dockerfile

```diff
@@ -68,7 +68,7 @@ WORKDIR /home/learn-ocaml
 
 COPY --from=compilation /home/opam/install-prefix /usr
 
-ENTRYPOINT ["dumb-init","learn-ocaml","--sync-dir=/sync","--repo=/repository"]
+ENTRYPOINT ["dumb-init","/usr/bin∕learn-ocaml","--sync-dir=/sync","--repo=/repository"]
```

Comment:
Hi @AltGr, I believe I understood why the CI failed (it seems a weird but very funny explanation :)
The 3rd slash in this diff is not a true slash:

![2021-07-29_21-59-24_Screenshot_fakeslash](https://user-images.githubusercontent.com/10367254/127558031-b542af14-c1d8-4249-8ab6-bd6655260bb8.png)

So the fix should just be:
```suggestion
ENTRYPOINT ["dumb-init","/usr/bin/learn-ocaml","--sync-dir=/sync","--repo=/repository"]
```
(feel free to force-push)

Reply:
Indeed! I finally found it before seeing your message :sweat_smile: how could that ever have happened ?
As mentionned in the first comment, the Docker containers weren't fixed yet, but now it should be ok.

Last CI failure is because https://github.com/ocaml-sf/learn-ocaml-corpus/pull/32 would need merging first (but in its current state it would break the corpus on released learn-ocaml :/ )

Reply:
> how could that ever have happened ?

I think that's because this character may be associated to a keybinding very close to <kbd>Shift+/</kbd>:
with my keyboard config, it is <kbd>AltGr+Shift+/</kbd>

> Last CI failure is because ocaml-sf/learn-ocaml-corpus#32 would need merging first

OK, thanks for opening this other PR!

> (but in its current state it would break the corpus on released learn-ocaml :/ )

Oh, that's unfortunate… would you have ideas to overcome this? I mean:

* Keeping the whole learn-ocaml-corpus run within learn-ocaml's CI;
* Make it pass for both learn-ocaml.0.12 and learn-ocaml@master ?